tools/pkg-helpers/pytorch_pkg_helpers/__main__.py

```diff
@@ -42,7 +42,10 @@ def parse_args() -> argparse.Namespace:
         "--platform",
         help="Platform to generate for",
         type=str,
-        default=sys.platform,
+        default= (
+            sys.platform if os.getenv("PLATFORM", sys.platform) == ""
+            else os.getenv("PLATFORM", sys.platform)
```

Comment:
Should `default=os.getenv("PLATFORM", sys.platform)` be good enough?  As the latter is the default value if `env["PLATFORM"]` is empty

Reply:
I think it defaults to sys.platform if PLATFORM  is not set, if PLATFORM is set to empty string it will set it to empty string.